### PR TITLE
Update for current implementation of max_connections

### PR DESCRIPTION
**Pull request by janebeckman**
Revisions were made to max_connections and seg_max_connections, especially with new defaults. This updates to reflect that usage

### CODE REVIEW

File: markdown/clientaccess/client_auth.html.md.erb [hunk 1]
```diff
@@ -99,27 +99,25 @@ This example shows how to edit the `pg_hba.conf` file of the master to allow rem
 
 HAWQ allocates some resources on a per-connection basis, so setting the maximum number of connections allowed is recommended.
 
-To limit the number of active concurrent sessions to your HAWQ system, you can configure the `max_connections` server configuration parameter on master or the `seg_max_connections` server configuration parameter on segments. These parameters are *local* parameters, meaning that you must set them in the `hawq-site.xml` file of all HAWQ instances.
+To limit the number of active concurrent sessions to your HAWQ system, you can configure the `max_connections` server configuration parameter. This parameter is a *local* parameter, meaning that you must set them in the `hawq-site.xml` file of all HAWQ instances.
```
Comment (radarwave) [inline markdown/clientaccess/client_auth.html.md.erb]: The previous one seems better:
To limit the number of active concurrent sessions to your HAWQ system, you can configure the `max_connections` server configuration parameter on master or the `seg_max_connections` server configuration parameter on segments.

Except that I think we don't need to mention it's a *local* parameter. We don't need to set it in 'hawq-site.xml' unless we want one specify node to be different than others.

File: markdown/clientaccess/client_auth.html.md.erb [hunk 2]
```diff
@@ -99,27 +99,25 @@ This example shows how to edit the `pg_hba.conf` file of the master to allow rem
...
 
-When you set `max_connections`, you must also set the dependent parameter `max_prepared_transactions`. This value must be at least as large as the value of `max_connections`, and all HAWQ instances should be set to the same value.
+When you set `max_connections`, you can also set the value for the maximum concurrent connections for the segments by using `seg_max_connections`. Setting this parameter overrides the value of `max_connections` on the segment instances.
```
Comment (radarwave) [inline markdown/clientaccess/client_auth.html.md.erb]: 'seg_max_connections' does not overwrite 'max_connections', they are separate GUCs. 

'max_connections' will only affect master node's max connections.
'seg_max_connections' only affect segment nodes's max connections.

File: markdown/clientaccess/client_auth.html.md.erb [hunk 3]
```diff
@@ -99,27 +99,25 @@ This example shows how to edit the `pg_hba.conf` file of the master to allow rem
...
 
 Example `$GPHOME/etc/hawq-site.xml` configuration:
 
 ``` xml
   <property>
       <name>max_connections</name>
-      <value>500</value>
+      <value>200</value>
```
Comment (radarwave) [inline markdown/clientaccess/client_auth.html.md.erb]: It's ok, but I think better to be 1280 as the default value we used.

File: markdown/clientaccess/client_auth.html.md.erb [hunk 4]
```diff
@@ -99,27 +99,25 @@ This example shows how to edit the `pg_hba.conf` file of the master to allow rem
...
   </property>
   <property>
-      <name>max_prepared_transactions</name>
+      <name>seg_max_connectionss</name>
```
Comment (radarwave) [inline markdown/clientaccess/client_auth.html.md.erb]: As I mentioned above, this should better to be 3000 as our default value.

**Comment by radarwave** [comment]
@jiny2 @yaoj2 

Please help to review since you made the last change of these two GUCs.

Appending the related jira and commits:
https://issues.apache.org/jira/browse/HAWQ-367
https://github.com/apache/incubator-hawq/pull/301

https://issues.apache.org/jira/browse/HAWQ-606

commit 1eaa931e3f4704e61fe7c819db1921b259c27df3
Author: yaoj2 jyao@pivotal.io
Date: Thu Mar 31 10:56:03 2016 +0800
remove gp_enable_column_oriented_table and change default seg_max_connections

File: markdown/clientaccess/client_auth.html.md.erb (hunk 3 above)
Comment (jinyismilodon) [reply markdown/clientaccess/client_auth.html.md.erb]: I find we mentioned the best practice setting is 1280, so I agree with Radar if we can directly set it to 1280 to reflect what we recommended.

Comment (jiny2) [reply markdown/clientaccess/client_auth.html.md.erb]: I am ok with default value changes, as Radar said, if we mentioned the best practice value as 1280 for this guc, maybe it is better to directly change to 1280 to reflect what we recommended. Thanks.

File: markdown/clientaccess/client_auth.html.md.erb (hunk 4 above)
Comment (jiny2) [reply markdown/clientaccess/client_auth.html.md.erb]: As above.

**Comment by janebeckman** [comment]
Do we need to make any mention of minimum values for either guc?

**Comment by jiny2** [comment]
Personally, I think no. @janebeckman  Thanks.

**Comment by radarwave** [comment]
@janebeckman I think there is no need to mention the minimum values.

**Pull request closed without merging**